packages/operator/pkg/errors/errors.go

```diff
@@ -107,3 +108,31 @@ func IsSpecWasTouchedError(err error) bool {
 	_, ok := err.(SpecWasTouched)
 	return ok
 }
+
+
+type DeletingServiceHasJobs struct {
+	// ID of BatchInferenceService
+	Entity string
+}
+
+func (e DeletingServiceHasJobs) Error() string {
+	return fmt.Sprintf(`Unable to delete service: "%s". Cause: there are child jobs`, e.Entity)
+}
+
+func (e DeletingServiceHasJobs) HTTPCode() int {
```

Comment:
Do you want this mechanism to replace an error converter? This method is not used yet, right?

Reply:
I deleted this dead code for now, but yes, I would like to replace our error converter mechanism by changing
 [https://github.com/odahu/odahu-flow/blob/516190cbea4588fd9cb894447ca68fc49eff0530/packages/operator/pkg/errors/httpcode.go#L8](CalculateHTTPStatusCode) next way

```
type HTTPCodeAware interface {
	HTTPCode() int
}

func CalculateHTTPStatusCode(err error) int {

	httpAwareErr, ok := err.(HTTPCodeAware)
	if ok {
		return httpAwareErr.HTTPCode()
	}

	return http.StatusInternalServerError

}
```

Let's discuss it and if it's okay do it in a separate PR